nservicebus/upgrades/asb-6to7.md

```diff
@@ -11,35 +11,43 @@ related:
 
+When selecting `EndpointOrientedTopology`, it is also necessary to configure [publisher names](/nservicebus/azure-service-bus/publisher-names-configuration.md), in order to ensure that subscribers are subscribed to the correct publisher:
 
-### Setting The Connection String
+snippet:publisher_names_mapping_upgrade_guide
 
-Setting the connection string can still be done using the `ConnectionString` extension method:
+For more details on topologies refer to the [Azure Service Bus Transport Topologies](/nservicebus/azure-service-bus/topologies/) article.
 
-snippet:setting_asb_connection_string
+## Sanitization
 
+In Versions 6 and below sanitization was performed by default and the MD5 algorithm was used to truncate entity names. In Versions 7 and above, the sanitization has to be enabled and configured explicitly.
 
-### Setting Entity Property Values
+In order to maintain backward compatibility, [register a custom sanitization strategy](/nservicebus/azure-service-bus/sanitization.md#backward-compatibility-with-versions-6-and-below).
 
-The other configuration properties found on `AzureServiceBusQueueConfig` were related to the configuration of Azure Service Bus entities. Even though the name implied differently, these settings were not only applied to queues, but also to topics and subscriptions.
+In version 6.4.0 `NamingConventions` class was introduced to customize sanitization. The class is obsoleted. Instead, implement a [custom sanitization strategy](/nservicebus/azure-service-bus/sanitization.md#version-7-and-above).
 
-In the new configuration API the settings for queues, topics and subscriptions can be configured individually using the `Queues()`, `Topics()` and `Subscriptions()` extension points.
 
-For example the lock duration on a queue can be configured using the `LockDuration` setting:
+## New Configuration API
 
-snippet:setting_queue_properties
+In Versions 6 and below the Azure Service Bus transport was configured using an XML configuration section called `AzureServiceBusQueueConfig`. This section has been removed in favor of a more granular, code based configuration API.
 
-and the size of the topics can be configured using the `MaxSizeInMegabytes` setting:
+The new configuration API is accessible through extension methods on the `UseTransport<AzureServiceBusTransport>()` extension point in the endpoint configuration. Refer to the [Full Configuration Page](/nservicebus/azure-service-bus/configuration/full.md) for more details.
+
+snippet:AzureServiceBusTransportWithAzure
 
-snippet:setting_topic_properties
+
+### Setting The Connection String
+
+Setting the connection string can still be done using the `ConnectionString` extension method:
+
+snippet:setting_asb_connection_string
```

Comment:
i notice u use a non upgrade specific snippet here. This is not ideal since when u add a version 8 of setting_asb_connection_string that will show up in the version 7 upgrade guide. it is better to create specific snippets for the upgrade use case. so eg a new snippet named `6to7-setting_asb_connection_string`